platform/android/audio_driver_opensl.cpp

```diff
@@ -317,7 +317,7 @@ Error AudioDriverOpenSL::capture_stop() {
 
 int AudioDriverOpenSL::get_mix_rate() const {
 
-	return 44100;
+	return DEFAULT_MIX_RATE;
```

Comment:
Same here, if that was hardcoded as the only value supported by Android, this should be commented.

Reply:
Android is using only 44100, it's selected with the `SL_SAMPLINGRATE_44_1`, so I think that the hardcoded `44100` is more correct here.

Reply:
Pushed an update going back to `44100` with an extra comment.